src/Components/Home/Searchbar.js

```diff
@@ -15,6 +15,9 @@ const Searchbar = ({ searchHandler, searchTerm }) => {
           name="user"
           id="search-input"
           placeholder="Search user..."
+          style={{
```

Comment:
I'm not sure that this style makes any difference. I've tried it without and can't see that it does anything.

Reply:
Removed the style.